code/modules/projectiles/gun_attachables.dm

```diff
@@ -759,15 +759,15 @@ inaccurate. Don't worry if force is ever negative, it won't runtime.
 	flags_attach_features = ATTACH_ACTIVATION
 	attach_icon = "none"
 	zoom_offset = 3
-	zoom_viewsize = 7
+	zoom_viewsize = 0
 
 
 /obj/item/attachable/scope/unremovable/tl102
 	name = "TL-102 smart sight"
 	desc = "An unremovable smart sight built for use with the tl102, it does nearly all the aiming work for the gun's integrated IFF systems."
 	attach_icon = "none"
 	zoom_offset = 3
-	zoom_viewsize = 7
```

Comment:
Why are you making zoom 0 here everywhere

Reply:
because they were 7 because 7 = 7 tiles to each side of view + 1 of center, now this doenst make sense anymore and even less with widescreen if it comes for the future, this just instead tells how much tiles it will increase.